Cargo.toml

```
@@ -16,8 +16,24 @@ rust-version = "1.70"
 default = ["std"]
 std = ["kurbo/std"]
 libm = ["kurbo/libm"]
+serde = ["smallvec/serde", "kurbo/serde", "dep:serde_bytes", "dep:serde"]
+
+[package.metadata.docs.rs]
+features = ["serde"]
 
 [dependencies]
 # NOTE: Make sure to keep this in sync with the version badge in README.md
 kurbo = { version = "0.11.0", default-features = false }
 smallvec = "1.13.2"
+
+[dependencies.serde]
+version = "1.0.105"
+optional = true
+default-features = false
+features = ["alloc", "derive"]
+
+[dependencies.serde_bytes]
+version = "0.11.12"
```

I know these versions match what Kurbo has, but let's specify the requirements in `Cargo.toml` so that they match what is in `Cargo.lock` and thus what we test. This policy was adopted as part of [rfcs#5](https://github.com/linebender/rfcs/blob/main/rfcs/0005-version-matrix.md#cargotoml-matches-cargolock). Kurbo will eventually do so too.

```suggestion
[dependencies.serde]
version = "1.0.197"
optional = true
default-features = false
features = ["alloc", "derive"]

[dependencies.serde_bytes]
version = "0.11.14"
```